patterns/stateful/README.md

```diff
@@ -129,6 +135,45 @@ The following command will update the `kubeconfig` on your local machine and all
     /dev/nvme1n1    24594768 2886716  20433380  13% /run/containerd
     ```
 
+## Validate Velero Install
```

Comment:
Lets remove lines 138-147, and then continue the numbering from above (make it one continuous block of validation steps)

Reply:
Consolidated items per your recommendation with latest push